presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java

```diff
@@ -64,19 +80,21 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
     }
 
     /**
-     * Returns the coordinator's uri.
+     * Returns an object with information about coordinator's URI.
      *
-     * @return an Optional {@link CoordinatorLocation}.
+     * @return an object with information about coordinator's URI
```

Comment:
Can maintain the information it returns an Optional

Reply:
added